hbase-zookeeper/pom.xml

```diff
@@ -139,6 +139,15 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- ZooKeeperServer needs the below dependencies, thus MiniZooKeeperCluster also needs them -->
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
```

Comment:
This undeclared dependency of recent zookeeper versions is annoying. We will CNFE without it.

Reply:
Yes, I think a better way is to have a zookeeper-client module and a zookeeper-server module. Now they only have a zookeeper module but they only declare the dependencies for zookeeper-client at compile scope so we need to manually add the zookeeper server dependencies...